website/docs/r/container_cluster.html.markdown

```diff
@@ -53,34 +53,41 @@ resource "google_container_cluster" "primary" {
 
 ## Argument Reference
 
-* `initial_node_count` - (Required) The number of nodes to create in this
-    cluster (not including the Kubernetes master).
-
 * `name` - (Required) The name of the cluster, unique within the project and
     zone.
 
 * `zone` - (Required) The zone that the master and the number of nodes specified
     in `initial_node_count` should be created in.
 
 - - -
-* `master_auth` - (Optional) The authentication information for accessing the
-    Kubernetes master.
 
-* `additional_zones` - (Optional) If additional zones are configured, the number
-    of nodes specified in `initial_node_count` is created in all specified zones.
+* `additional_zones` - (Optional) The list of additional Google Compute Engine
+    locations in which the cluster's nodes should be located. If additional zones are
+    configured, the number of nodes specified in `initial_node_count` is created in
+    all specified zones.
 
 * `addons_config` - (Optional) The configuration for addons supported by Google
-    Container Engine
+    Container Engine. Structure is documented below.
 
 * `cluster_ipv4_cidr` - (Optional) The IP address range of the container pods in
     this cluster. Default is an automatically assigned CIDR.
 
 * `description` - (Optional) Description of the cluster.
 
+* `enable_legacy_abac` - (Optional) Whether the ABAC authorizer is enabled for this cluster.
+    When enabled, identities in the system, including service accounts, nodes, and controllers,
+    will have statically granted permissions beyond those provided by the RBAC configuration or IAM.
+
+* `initial_node_count` - (Required) The number of nodes to create in this
```

Comment:
Why is this grouped with the Optional fields?

Reply:
Because it's actually Optional- updated the docs :)